src/icons/item-icon/ItemIcon.tsx

```diff
@@ -39,6 +40,7 @@ import IconFolderPersonal from '../../icon/content/FolderPersonal32';
 const itemIconTable = {
     audio: IconFileAudio,
     bookmark: BookmarkIcon,
+    boxcanvas: IconFileCanvas,
```

Comment:
should this be just `canvas` instead of `boxcanvas`?

the other icon names that are `IconFileName` have the pattern:
`name: IconFileName`

```suggestion
    canvas: IconFileCanvas,
```

Reply:
@jstoffan I think that would be good pattern to use as that's how we handle the Google icons and it makes it clear how the icon "belongs" to a specific brand.

@pjzabinski if you decide to incorporate Jared's suggestion (`IconFileCanvas` ==> `IconFileBoxCanvas`), we should remember to make the same name change across the other files: `src/icons/file-icon/FileIcon.tsx`, `src/components/preview/previewIcons.ts`, etc.

Also would mean you would have to re-sort some of the those usages (sorry bout that!)